charts/rancher-webhook/rancher-webhook/1.0.0+up0.2.0-beta04/values.yaml

```diff
@@ -12,4 +12,10 @@ capi:
   enabled: false
 
 mcm:
-  enabled: true
\ No newline at end of file
+  enabled: true
+
+preDelete:
+  enabled: true
+  image:
+    repository: rancher/kubectl
+    tag: v0.2.0-beta04
```

Comment:
This isn't correct, looks like it grabbed the webhook tag, which is odd since it looks correct in the chart: https://github.com/rancher/webhook/blob/master/charts/rancher-webhook/values.yaml#L21

https://hub.docker.com/r/rancher/kubectl/tags?page=1&ordering=last_updated

Reply:
@cbron if you download the tgz file and look at the file contents, it does show `v0.2.0-beta04`

![Screen Shot 2021-08-18 at 10 51 54 AM](https://user-images.githubusercontent.com/13127443/129947600-a548efa3-2a47-4ddc-ace6-61c30e926742.png)

Reply:
This is probably because we replace all tags with the HELM_TAG here

https://github.com/rancher/webhook/blob/efe9b4b5743347554cc6c332d1b615de4c321b66/scripts/package-helm#L20-L22

Where the HELM_TAG is set to have the RC based on:

https://github.com/rancher/webhook/blob/245c6bc9362b0204c3bcad7c923ade36087ea055/scripts/version#L22

Reply:
Thank you for the investigation. I will fix it in rancher/webhook